pkg/minishift/registration/redhat.go

```diff
@@ -90,7 +89,28 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 				param.Username,
 				minishiftStrings.EscapeSingleQuote(param.Password))
 
-			fmt.Print("   Registration in progress ")
+			// prepare docker login command for registry.redhat.io
+			dockerLoginCommand := fmt.Sprintf("docker login --username %s --password %s %s",
```

Comment:
Not a blocker to merge this PR, but for readability or scalability, this string can be extracted as: 
```
dockerLoginTmpl = "docker login --username %s --password %s %s"
dockerLoginCommand := fmt.Sprintf(dockerLoginTmpl, param.Username, ......)
```